sda-svc/test/accession.py

```diff
@@ -83,16 +83,16 @@
                           "type": "md5",
                           "value": "%s"
                          }
-                        ] 
+                        ]
 }
 """ % (args.user,
-       args.archivepath,
+       args.inboxpath,
        args.accessionid,
        args.decsha256,
        args.decmd5)
 
 channel.basic_publish(exchange=exchange,
-                      routing_key= 'stableIDs',
+                      routing_key= 'files',
```

Comment:
I guess this won't work without intercept?

Reply:
Changed to `accessionIDs`